_overviews/toolkit/web-server-input.md

```diff
@@ -0,0 +1,232 @@
+---
+title: How to handle user input?
+type: section
+description: Handling user input with Cask
+num: 34
+previous-page: web-server-query-parameters
+next-page: web-server-websockets
+---
+
+{% include markdown.html path="_markdown/install-cask.md" %}
+
+## Handling form-encoded input
+
+Similarly to path segments and query parameters, form fields are read by using endpoint method arguments. Use `cask.postForm`
+annotation and set the HTML form method to `post`.
+
+In this example we create a form asking for name and surname of a user and then redirect to a greeting page. Notice the
+use of `cask.Response`. The default returned content type is `text/plain`, set it to `text/html` in order for browser to display
+the form correctly.
```

Comment:
It feels like we should have introduced this `cask.Response` subtlety before. We're mixing two issues here, it seems.

Reply:
I wonder if I should add a part dedicated specifically to this.